### ISSUE TITLE
We stole your patterns

### ISSUE THREAD

**Issue by pudo** [codeforamerica/civic-tech-patterns#42]
Hey guys, @lucyfedia and I have mad a push yesterday to merge all of the really cool points from this repo into it's sister project, CivicPatterns.org: https://github.com/Code-for-All/civicpatterns.org/pull/20. It would be great to know if you think that we've done your notions justice, and if we've found the right matches where the same pattern exists in both lists. What do you think?

**Comment by bensheldon** [comment]
+:100: for doing the patterns justice in your *much* better presentation. I can comment on the actual comment itself in the https://github.com/Code-for-All/civicpatterns.org/pull/20 pull request. But I'm :+1: in favor of merging the 2 repos and I think yours wins on both content and style. 

My big request is that you credit all of the contributors to this civic-tech-patterns repo in your Authors file, both contributors in code as well as those participating in the Issues. This was definitely the work of many, many people. I can pull that list together and make a pull request to your repo.

There is also a few un-committed patterns in the Issues and PRs. I'd like to get those contributed as well. Basically close-out this repo and link it to yours as the canonical location. 

:heart_eyes_cat:

**Comment by pudo** [comment]
Yes for credit! We've already tried to add [credits in a general sense](https://github.com/pudo/civicpatterns.org/commit/b83102ef0c661e2819cf056a762477630f6994c7#diff-6fc4f01eb8f7f63232808e31268c4430R13) but I would really like to also get in the names of individuals (not least to avoid this becoming the product of any particular org ;)). So a list would be much appreciated!

I'll have a sift through your issues and start including them as well, very good thing to mention!

**Comment by louh** [comment]
First I've seen civicpatterns.org, just wanted to :+1: the clean presentation! Looking forward to seeing it grow.

**Comment by pudo** [comment]
... ping @montselobos, praising your beautiful theme :)
